decidim-core/lib/decidim/core.rb

```diff
@@ -270,9 +270,20 @@ def self.seed!
     false
   end
 
-  # How long can a user remained logged in before the session expires
+  # How long can a user remained logged in before the session expires. Notice that
+  # this is also maximum time that user can idle before getting automatically signed out.
   config_accessor :expire_session_after do
-    1.day
+    30.minutes
+  end
+
+  # If set to true, users have option to "remember me".
```

Comment:
Add explanation that the user won't be forced to log out in case they have chosen the "remember me" option in the login form. In other words, `expire_session_after` won't take effect when the user wants to be remembered.